accounts-db/src/accounts_index.rs

```diff
@@ -1219,7 +1219,7 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> AccountsIndex<T, U> {
             .unwrap_or(0)
     }
 
-    fn update_spl_token_secondary_indexes<G: solana_inline_spl::token::GenericTokenAccount>(
+    fn update_spl_token_secondary_indexes<G: spl_generic_token::token::GenericTokenAccount>(
```

Comment:
@brooksprumo we talked about this in dms and agreed its fine from a consensus perspective since it only affects rpc but if you could sign off on this

specifically, the new token account parser is more strict and requires token accounts to have been initialized (ie initialized or frozen states). the changed tests show the effect this has. this is a straightforward bugfix, so this is correct to do, and correct to do with no feature gate if the secondary indexes are not used for runtime account loading